spec/spec_helper.rb

```diff
@@ -0,0 +1,66 @@
+ROOT = Pathname.new(File.expand_path("../", __dir__))
+
+require ROOT.join("lib", "jekyll.rb")
```

Comment:
`require_relative "../lib/jekyll"`, however "jekyll" should already be in the path.